server/src/main/java/org/elasticsearch/threadpool/ThreadPool.java

```diff
@@ -182,7 +182,10 @@ public ThreadPool(final Settings settings, final ExecutorBuilder<?>... customBui
         final int halfProcMaxAt5 = halfAllocatedProcessorsMaxFive(allocatedProcessors);
         final int halfProcMaxAt10 = halfAllocatedProcessorsMaxTen(allocatedProcessors);
         final int genericThreadPoolMax = boundedBy(4 * allocatedProcessors, 128, 512);
-        builders.put(Names.GENERIC, new ScalingExecutorBuilder(Names.GENERIC, 4, genericThreadPoolMax, TimeValue.timeValueSeconds(30)));
+        builders.put(
+            Names.GENERIC,
+            new ScalingExecutorBuilder(Names.GENERIC, 4, genericThreadPoolMax, TimeValue.timeValueSeconds(30), false)
```

Comment:
Could we move these values (4 and 30) to constants please?

Reply:
I agree with Henning here.